src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationUpdateCommandTest.php

```diff
@@ -37,6 +37,17 @@ public function testDumpMessagesAndCleanWithDeprecatedCommandName()
         $this->assertMatchesRegularExpression('/1 message was successfully extracted/', $tester->getDisplay());
     }
 
+    public function testDumpBlankMessagesAndCleanWithDeprecatedCommandName()
+    {
+        $tester = $this->createCommandTester(['messages' => ['foo' => '']]);
+        if (!method_exists($tester, 'blank')) {
+            $this->markTestSkipped('The --blank option is not supported by this extractor');
+        }
```

Comment:
Not sure if thats the best approach to handle failing tests in low deps jobs, but I think its reasonable :)

Reply:
Seems like they are always skipped now... That was not intended.

Reply:
It seems to work as intended in CI. Tests are run in the `8.1, high-deps` job, but skipped in `8.1, low-deps`.

Reply:
Thinking about it, it would actually be better to assert the output based on that condition. So normal output if method exists and error message if not. Will change that.